docs/_user_guide/best_practices_matlab_slurm_job.md

```diff
@@ -1,33 +1,33 @@
 ## Matlab Example Slurm Script
 The simplest Matlab script looks like:
 
-```matlab
+```matlab title="Matlab Code"
 disp('Hello world')
 ```
 
 Save this line to a new file called `hello_world.m`. 
 
-## Available Matlab versions on Yen10
+## Available Matlab versions on `Yen10`
 Several version of Matlab are installed on the yens and the default version is designated with `(D)`:
 
-```bash
+```bash title="Terminal Command"
 module avail matlab
 ```
 
```

Comment:
It would be good here to show the example output of running `module avail matlab` to show all the versions:

```
--------------------------------------------------------------- Global Aliases ---------------------------------------------------------------


----------------------------------------------------------- /software/modules/Core -----------------------------------------------------------
   matlab/R2018a    matlab/R2018b    matlab/R2019b    matlab/R2021b    matlab/R2022a    matlab/R2022b (D)    matlab/R2024a
```